src/core/lightning/index.ts

```diff
@@ -293,19 +298,40 @@ export const LightningMixin = (superclass) =>
             })
           }
 
-          await lockExtendOrThrow({ lock, logger: lightningLoggerOnUs }, async () => {
-            const tx = await ledger.addOnUsPayment({
-              description: memoInvoice,
-              sats,
-              metadata,
-              payerUser: this.user,
-              payeeUser,
-              memoPayer,
-              shareMemoWithPayee: isPushPayment,
-              lastPrice: UserWallet.lastPrice,
-            })
-            return tx
-          })
+          const price = await PriceService().getCurrentPrice()
+          if (price instanceof Error) throw price
+          const lnFee = toSats(0)
+          const usd = sats * price
+          const usdFee = lnFee * price
+
+          const payerWallet = await WalletsRepository().findById(this.user.id)
+          if (payerWallet instanceof CouldNotFindError) throw payerWallet
+          if (payerWallet instanceof Error) throw payerWallet
+          const recipientWallet = await WalletsRepository().findById(payeeUser.id)
+          if (recipientWallet instanceof CouldNotFindError) throw recipientWallet
+          if (recipientWallet instanceof Error) throw recipientWallet
+
+          const journal = await lockExtendOrThrow(
+            { lock, logger: lightningLoggerOnUs },
+            async () => {
+              return LedgerService().sendLnIntraledgerTx({
```

Comment:
same here, we are not `sending` we are adding a record to the ledger

Reply:
This make sense. Change all send/receive references in the LedgerService to `add...`: https://github.com/GaloyMoney/galoy/pull/582/commits/e03d7bee78d2c789f9baece3cfd03ac695c425cf